packages/google_maps_flutter/google_maps_flutter_web/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 0.5.10+1
 
+* Deprecate zIndex parameter in Marker and replace with zIndexInt.
```

Comment:
I'd expect to see a change here for the web?

* https://github.com/flutter/packages/blob/79f8b0b6655132ac8d11a50243afe85867a09405/packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart#L269

Reply:
Thank you for the reminder, didn't think about web. I can just add that in right away